### DISCUSSION TITLE
Help with creation of own icon

### DISCUSSION THREAD

**Discussion by archipita** [elax46/custom-brand-icons#841] [General]
HI,
I have created a new icon using Inkscape, following the instructions (size, color, etc.).
I have then added the icon data in the js file, reloaded it, but I cannot see the icon listed in HA (phu:...).
I have already tried to clear the cache and to restart HA.
Do I miss anything?
How can I check if the created icon is OK for HA?
Thanks.

**Comment by elax46** [comment]
hi if you want to integrate a new icon you can either make a request or open a request and tell us the type of icon
